src/cli/cluster/cluster_manager.js

```diff
@@ -102,8 +103,14 @@ export default class ClusterManager {
 
     if (opts.watch) {
       const pluginPaths = config.get('plugins.paths');
-      const scanDirs = config.get('plugins.scanDirs');
-      const extraPaths = [...pluginPaths, ...scanDirs];
+      const scanDirs = [
+        ...config.get('plugins.scanDirs'),
+        resolve(REPO_ROOT, 'src/plugins'),
```

Comment:
Does `x-pack/plugins` also need to go here?

Reply:
Yeah, I suppose so